### ISSUE TITLE
Mute not working for SMB videos

### ISSUE THREAD

**Issue by cosmoxl** [theothernt/AerialViews#132]
https://old.reddit.com/r/ShieldAndroidTV/comments/1afg2uw/aerial_views_screensaver_not_muting_smb_videos/

Moving this reddit conversation here at the request of theothernt to better track the problem.  Thanks for looking into it.

**Comment by theothernt** [comment]
Thanks for creating the issue. 

This bug is odd but it's also the 2nd bug related to the Shield recently. Both are simple playback issues that shouldn't happen.

What I'd like to try is to give you an old debug version of the app, maybe from a year ago, and see if that works. If it does, we can go from there and narrow down the issue. The debug version of the app will not override the Play Store version you have installed.

**Comment by cosmoxl** [comment]
> Thanks for creating the issue.
> 
> This bug is odd but it's also the 2nd bug related to the Shield recently. Both are simple playback issues that shouldn't happen.
> 
> What I'd like to try is to give you an old debug version of the app, maybe from a year ago, and see if that works. If it does, we can go from there and narrow down the issue. The debug version of the app will not override the Play Store version you have installed.

sounds good

**Comment by theothernt** [comment]
Ok, here is [1.4.3-debug](https://www.dropbox.com/scl/fi/90q6sxnb92yn8dsjehfz3/aerial-views-1.4.3-debug.apk?rlkey=q2bubwb5pe0bee1k7lkyv54ql&dl=0) - let me know if this has the same issue

**Comment by cosmoxl** [comment]
> Ok, here is [1.4.3-debug](https://www.dropbox.com/scl/fi/90q6sxnb92yn8dsjehfz3/aerial-views-1.4.3-debug.apk?rlkey=q2bubwb5pe0bee1k7lkyv54ql&dl=0) - let me know if this has the same issue

Yes, same problem with audio playing though mute option is ticked.  This version also has another thing I'd noticed in the recent release and that's that more often than not there's corruption of the video - either a strip on the right side or a strip on the bottom.  Seems like bottom for 4:3 aspect ratio videos, and right side corruption for videos wider than 16:9.

**Comment by cosmoxl** [comment]
Sorry, no difference with the 1.6.3-debug with regard to the audio playing.

**Comment by theothernt** [comment]
> Sorry, no difference with the 1.6.3-debug with regard to the audio playing.

Ok, he is [another version](https://www.dropbox.com/scl/fi/uqh17scp828uxgpdjqoo4/aerial-views-1.6.3-debug2.apk?rlkey=ny9rur4zphqbnk4111dm3ru6h&dl=0) - this time it disables the audio track inside the video, so the volume doesn't matter.

Also, I did just think of something. Is your Shield directly connected to your TV or does the audio go through another device?

**Comment by cosmoxl** [comment]
> > Sorry, no difference with the 1.6.3-debug with regard to the audio playing.
> 
> Ok, he is [another version](https://www.dropbox.com/scl/fi/uqh17scp828uxgpdjqoo4/aerial-views-1.6.3-debug2.apk?rlkey=ny9rur4zphqbnk4111dm3ru6h&dl=0) - this time it disables the audio track inside the video, so the volume doesn't matter.
> 
> Also, I did just think of something. Is your Shield directly connected to your TV or does the audio go through another device?

The shield is connected to an AVR then to the TV.  The good news is that this did the trick for "muting" :D

**Comment by theothernt** [comment]
Great, that's another bug fixed. It'll be in the next release but that's a few weeks away, at least.

Next is the video corruption you're seeing. Is this only with your custom videos?

**Comment by cosmoxl** [comment]
> Great, that's another bug fixed. It'll be in the next release but that's a few weeks away, at least.
> 
> Next is the video corruption you're seeing. Is this only with your custom videos?

yeah, and I see the same corruption in just video player.  it's due to the cropping to fill the screen, which is sort of necessary for a screen saver.  So for widescreen it crops the sides, for 4:3 it crops the top and bottom.

**Comment by theothernt** [comment]
Would it help to be able to turn off the crop? Just Player allows you to toggle this while playing to test if it helps or not.

**Comment by cosmoxl** [comment]
> Would it help to be able to turn off the crop? Just Player allows you to toggle this while playing to test if it helps or not.

I think the crop to fill the screen is necessary but some people might like the option.  The point of the screensaver is still being accomplished even if it's not pretty along the bottom or side. :)

**Comment by theothernt** [comment]
Version 1.7 has been released with this fix on GitHub, it'll be on the Play Store in a few days.
